scripts/makedeb.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+mkdir -p lazygit_deb/{DEBIAN,usr/local/bin}
+cat << EOF > lazygit_deb/DEBIAN/control
+Package: lazygit
+Version: 0.44.1
```

Comment:
Is this version necessary? I'd prefer to not have to bump this every time we cut a new release